token/cli/src/main.rs

```diff
@@ -2603,15 +2628,27 @@ fn main() -> Result<(), Error> {
     };
 
     solana_logger::setup_with_default("solana=info");
+    let result = process_command(sub_command, matches, &config, wallet_manager, bulk_signers)
+        .map_err::<Error, _>(|err| DisplayError::new_as_boxed(err).into())?;
+    println!("{}", result);
+    Ok(())
+}
 
-    let result = match (sub_command, sub_matches) {
-        ("bench", Some(arg_matches)) => bench_process_command(
+fn process_command(
+    sub_command: &str,
+    sub_matches: &ArgMatches<'_>,
+    config: &Config,
+    mut wallet_manager: Option<Arc<RemoteWalletManager>>,
+    mut bulk_signers: Vec<Box<dyn Signer>>,
+) -> CommandResult {
+    match (sub_command, sub_matches) {
+        ("bench", arg_matches) => bench_process_command(
```

Comment:
I spent too long trying to figure out why this went from `Some(arg_matches)` to `arg_matches`... wow, our handling of matches/arg_matches/sub_matches was dumb. Thanks for cleaning it up!